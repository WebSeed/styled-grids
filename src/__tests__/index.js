import React from 'react'
import renderer from 'react-test-renderer'
import 'jest-styled-components'

import Grid from '..'

describe('styled-grids', () => {
  it('should match the snapshot', () => {
    const tree = renderer.create(<Grid />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
