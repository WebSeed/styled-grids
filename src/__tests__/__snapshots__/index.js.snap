// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`styled-grids should match the snapshot 1`] = `
.c0 {
  background-color: green;
}

<div
  className="c0"
/>
`;
